README.md

```diff
@@ -1,5 +1,6 @@
+# This library has been deprecated. 
 
-
+## Authenticating with an API key has been deprecated. Support for email/password authentication will be dropped in the future with authentication by cryptographic keys/wallets will be the only supported method of authentication.
```

Comment:
That sentence is a bit complicated, maybe:

```
Support for email/password authentication will be dropped in the future and cryptographic keys/wallets will be the only supported method.
```

Reply:
Much better, thanks @juhah